crates/turbopack-ecmascript-hmr-protocol/src/lib.rs

```diff
@@ -135,7 +135,7 @@ pub struct Issue<'a> {
     pub category: &'a str,
 
     pub title: &'a str,
-    pub description: &'a str,
+    pub description: StyledString,
```

Comment:
```suggestion
    pub description: &'a StyledString,
```

No need to clone that